collector/zfs.go

```diff
@@ -24,20 +24,16 @@ import (
 	"github.com/prometheus/common/log"
 )
 
-var zfsNotAvailableError = errors.New("ZFS / ZFS statistics are not available")
+var errorZFSNotAvailable = errors.New("ZFS / ZFS statistics are not available")
```

Comment:
Golint complained about `zfsNotAvailableError` and said it should be named `err...`. I just did it wrong. Somehow it didn't complain about the `error...` naming.